torch/ao/quantization/fx/utils.py

```diff
@@ -640,3 +641,220 @@ def get_skipped_module_name_and_classes(
         skipped_module_classes += get_custom_module_class_keys(prepare_custom_config.float_to_observed_mapping)
 
     return skipped_module_names, skipped_module_classes
+
+def _is_custom_module_lstm(node: Node, named_modules: Dict[str, torch.nn.Module]) -> bool:
+    """
+    Return whether this refers to the custom module LSTM flow.
+    """
+    if node.op != "call_module" or str(node.target) not in named_modules:
+        return False
+    mod = named_modules[str(node.target)]
+    return isinstance(mod, torch.nn.LSTM) or isinstance(mod, torch.ao.nn.quantizable.LSTM)
```

Comment:
`isinstance(mod, torch.nn.LSTM)` sounds too permissive, is there a way to restrict it to modules going through the custom module flow only?

Reply:
added a check for the qconfig and the qhandler